pages/index.tsx

```diff
@@ -211,23 +211,67 @@ function Index(): ReactNode {
 		const _errorCount = (
 			vaults
 				.filter((vault): boolean => {
+					const vaultData = aggregatedData.vaults[toAddress(vault.address)];
+					if (!vaultData) {
+						return false;
+					}
+
+					const riskScores = (vaultData.strategies ?? []).map((strategy): { strategy: { address: string; name: string; }; sum: number; isValid: boolean } => {
+						const {riskDetails} = strategy?.risk || {};
+						if (!riskDetails) {
+							return {strategy: {address: strategy.address, name: strategy.name}, sum: 0, isValid: false};
+						}
+						const sum = (
+							(riskDetails.TVLImpact || 0)
+							+ (riskDetails.auditScore || 0)
+							+ (riskDetails.codeReviewScore || 0)
+							+ (riskDetails.complexityScore || 0)
+							+ (riskDetails.longevityImpact || 0)
+							+ (riskDetails.protocolSafetyScore || 0)
+							+ (riskDetails.teamKnowledgeScore || 0)
+							+ (riskDetails.testingScore || 0)
+						);
+
+						return {
+							strategy: {
+								address: strategy.address,
+								name: strategy.name
+							},
+							sum,
+							isValid: sum > 0 && sum < 40
+						};
+					});
+
+					const descriptions = (vaultData.strategies ?? []).map((strategy): { strategy: any; isValid: boolean } => ({
+						strategy,
+						isValid: strategy.description !== ''
+					}));
+
 					const hasAnomalies = (
-						vault.strategies.length === 0
-						|| !aggregatedData.vaults[toAddress(vault.address)]?.hasValidPrice
-						|| !aggregatedData.vaults[toAddress(vault.address)]?.hasValidIcon
-						|| !aggregatedData.vaults[toAddress(vault.address)]?.hasValidTokenIcon
-						|| !aggregatedData.vaults[toAddress(vault.address)]?.hasLedgerIntegration
-						|| !aggregatedData.vaults[toAddress(vault.address)]?.hasValidStrategiesDescriptions
-						|| !aggregatedData.vaults[toAddress(vault.address)]?.hasValidStrategiesRisk
-						|| !aggregatedData.vaults[toAddress(vault.address)]?.hasYearnMetaFile
-						|| aggregatedData.vaults[toAddress(vault.address)]?.hasErrorAPY);
+						(appSettings.shouldShowStrategies && vault.strategies.length === 0)
+						|| (appSettings.shouldShowMissingTranslations && !!vaultData.missingTranslations)
+						|| (appSettings.shouldShowIcons && !vaultData.hasValidIcon)
+						|| (appSettings.shouldShowIcons && !vaultData.hasValidTokenIcon)
+						|| (appSettings.shouldShowPrice && !vaultData.hasValidPrice)
+						|| (appSettings.shouldShowRetirement && !vaultData.hasValidRetirement)
+						|| (appSettings.shouldShowYearnMetaFile && !vaultData.hasYearnMetaFile)
+						|| (appSettings.shouldShowLedgerLive && !vaultData.hasLedgerIntegration.deployed)
+						|| (appSettings.shouldShowStrategies && !vaultData.hasValidStrategiesDescriptions) // TODO: check if this is correct
+						|| (appSettings.shouldShowStrategies && !vaultData.hasValidStrategiesRisk) // TODO: check if this is correct
```

Comment:
@Majorfi do you know what these are?

Reply:
They don't have their `AnomaliesSection`, so are they even being used anywhere?

Reply:
> It's supposed to be in the `Want Token Description` group or in the `Risk` group (checkbox to filter)

Hmm `hasValidStrategiesDescriptions` should be in the `Want Token Description`? I don't get it